lib/utils.js

```diff
@@ -48,6 +48,8 @@ function getFileExtension({vendor, renderForCaching = false}) {
     case 'deepgram':
     case 'elevenlabs':
     case 'rimlabs':
+    case 'playht':
+    case vendor.startsWith('custom') ? vendor : 'cant_match_value':
```

Comment:
this seemingly deserves a comment

Reply:
fixed the code to make it clearer to read